v2/emailpassword/common-customizations/sessions/anonymous-session.mdx

```diff
@@ -123,6 +123,10 @@ else:
 - You could create your own application middleware which will inspects the request and auto adds a JWT to it in the response cookies. This way, whenever a user visits your website, and make an API call, they will get a JWT in their cookies, and you can use that JWT to track their activity.
 
 ## Verifying the JWT
+:::caution
+Please note that you **cannot** use the `verifySession` or `getSession` functions to verify JWTs from anonymous sessions. You will need to [manually verify the JWT](./with-jwt/jwt-verification).
```

Comment:
Specify why.. it's cause the verifySession / getSession check for the presence of certain claims in the JWT (like sessionHandle, refreshTokenHash etc) and those claims are only added by the Session recipe for logged in users, and will be missing from the anonymous JWT tokens.